packages/pyright-internal/src/common/pythonVersion.ts

```diff
@@ -76,7 +81,14 @@ export namespace PythonVersion {
         return true;
     }
 
-    export function isGreaterThan(version: PythonVersion, other: PythonVersion) {
+    export function isGreaterThan(version: PythonVersion | undefined, other: PythonVersion | undefined) {
+        if (version && !other) {
+            return true;
+        } else if (!version && other) {
```

Comment:
Nit: this "if" statement isn't needed. It's already covered by the "else if" statement below and returns the same result.

Reply:
Good point, will remove.